andela/ah-kifaru-frontend
# #166840879 user should be able to reset password

#### What does this PR do?
-  User can request for a password reset link then provide a new password.
#### Description of Task to be completed?
- Add a page for user to request passwords reset by providing their email address.
- Add a page where a user can reset their password by providing a new password.
#### How should this be manually tested?
- Clone the project then run command `npm run start:dev`
- Navigate to `localhost:3000/forgot-password` to request a password reset.
- An email is sent to the provided email address
- Visit the link in the mail to see the reset password page
#### Any background context you want to provide?
- none
#### What are the relevant pivotal tracker stories? (if applicable)
[#166840879](https://www.pivotaltracker.com/story/show/166840879)
